source/index.html

```diff
@@ -56,6 +55,67 @@ <h3 class="advantages__title">Заводит новых друзей</h3>
         </ul>
         <img src="https://via.placeholder.com/157x320" alt="" class="advantages__img">
       </section>
+      <section class="reviews slider">
+        <ul class="reviews__list">
+          <li class="reviews__item">
+            <h3 class="reviews__name">Николай Петров</h3>
+            <p class="reviews__text">«Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»</p>
+          </li>
+        </ul>
+      </section>
+      <section class="purchase slider">
+        <table class="purchase__table">
+          <tr class="purchase__row">
+            <th colspan="2" class="purchase__name">???</th>
+            <th colspan="2" class="purchase__name">Стандарт</th>
+            <th colspan="2" class="purchase__name">???</th>
+          </tr>
+          <tr class="purchase__row">
+            <td colspan="2" class="purchase__price">???</td>
+            <td colspan="2" class="purchase__price">3,99 USD</td>
+            <td colspan="2" class="purchase__price">???</td>
+          </tr>
+          <tr class="purchase__row">
+            <td class="purchase__content">Розовый фильтр</td>
+            <td class="purchase__included"></td>
+            <td class="purchase__content">Розовый фильтр</td>
+            <td class="purchase__included--not-included"></td>
```

Comment:
Модификатор без элемента.
Должно быть так:
```suggestion
            <td class="purchase__included purchase__included--not-included"></td>
```
В других ячейках тоже исправьте.